cpp/include/Ice/Protocol.h

```diff
@@ -5,8 +5,10 @@
 #ifndef ICE_PROTOCOL_H
 #define ICE_PROTOCOL_H
 
-#include <Ice/Config.h>
-#include <Ice/Version.h>
+#include "Config.h"
+#include "Ice/Version.h"
```

Comment:
Could this be:
```suggestion
#include "Version.h"
```

Reply:
No, before Version.h is generated from Version.ice. It's in a different directory.